lib/sync/types.go

```diff
@@ -44,3 +44,17 @@ type Fetcher interface {
 type Validator interface {
 	Validate(context.Context, *SyncInfo) error
 }
+
+type NodeInfo struct {
```

Comment:
Does watcher mode need it's own NodeInfo? We already have node.NodeInfo.

Reply:
In watcher mode, the basic policy of NodeInfo, we can use them from the NodeInfo of the connected validators.

Reply:
My intention is that watcher just wants to marshal what the watcher needs fields. (small fields).
But,  the basic policy of validators can be useful for client-based load balancing with watchers and clients.
But, Thie PR's watcher doesn't have an ability for distributed load balancing delivering txs ㅠㅠ.

Reply:
Anyway, I will change the `node.NodeInfo` with `watcher` , thanks @spikeekips :->